CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # Changes
 
-## Unreleased
+## 0.24.2 - 2025-02-24
```

Comment:
A lot of new features, but nothing breaking.  I grow weary of this pre-1.0 lifestyle.

Reply:
Indeed.